geoapps/driver_base/driver.py

```diff
@@ -96,7 +96,7 @@ def start(cls, filepath: str | Path, driver_class=None):
         print("Loading input file . . .")
         filepath = Path(filepath).resolve()
         ifile = InputFile.read_ui_json(
-            filepath, validations=driver_class._validations  # pylint: disable=W0212
+            filepath, validations=getattr(driver_class, "_validations", None)
```

Comment:
... looks like shoveling under the carpet. Disabling pylint is preferable as it is easier to track